DependencyInjection/FOSRestExtension.php

```diff
@@ -154,10 +154,6 @@ private function loadFormatListener(array $config, XmlFileLoader $loader, Contai
                 'fos_rest.format_listener.rules',
                 $config['format_listener']['rules']
             );
-
-            if ($config['view']['mime_types']['enabled'] && !method_exists(Request::class, 'getMimeTypes')) {
-                $container->getDefinition('fos_rest.format_negotiator')->addArgument($config['view']['mime_types']['formats']);
-            }
```

Comment:
Yeah the mime types are already registered by the mime type listener in the Request and the format negotiator fetch them from the request.